### PR TITLE
Include amp-validator and wait-on packages, setup amp-validator scripts

### LINKED ISSUE #442: AMP validation for all scenarios
**Is your feature request related to a problem? Please describe.**
After steps https://github.com/bbc/simorgh/issues/440 (creating amp routes) and https://github.com/bbc/simorgh/issues/441 (validating a single scenario) have been completed, we want to run the AMP validation script against all article scenarios. 

**Describe the solution you'd like**
Update the amp validation npm script to run against all of the article scenarios and output their results in the console. 

If there are any errors listed in the output, a separate issue should be opened to create an AMP-compliant component & container that addresses those points.

**Additional context**
See instructions here https://www.ampproject.org/docs/fundamentals/validate#command-line-tool and here https://www.npmjs.com/package/amphtml-validator

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #442 

This PR adds the amp-html validator cli and an npm run command to test validate the AMP HTML output of a single scenario.

The `wait-on` npm package has been used, and sometimes the AMP validator would run before the localhost server was available, causing it to immediately fail. `wait-on` will wait for the localhost server to be available before starting the amp-html-validator.

- [N/A] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by sareh** [comment]
@hindsc52 Looks good. You will need to include `wait-on` as a dependency. :)

**Review by pjlee11 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Simorgh",
   "main": "index.js",
   "scripts": {
+    "amp:validate": "wait-on http://localhost:7080/ && amphtml-validator http://localhost:7080/article/amp/scenario-25 || true",
```
Comment (sareh) [inline package.json]: To make the usage clearer it'd be good to add parentheses around the latter part`(amphtml-validator http://localhost:7080/article/amp/scenario-25 || true)`

File: package.json [hunk 2]
```diff
@@ -25,6 +26,7 @@
     "test": "npm run test:lint && npm run test:unit && npm run dataValidate",
     "test:accessibility": "npm run killApp && npm run build && run-p --race start bbcA11y",
     "test:accessibility:ci": "npm run build && run-p --race start bbcA11y",
+    "test:ampvalidator": "npm run killApp && npm run build && run-p --race start amp:validate",
```
Comment (sareh) [inline package.json]: If you could camelCase the name `test:ampvalidator` to `test:ampValidate` that would be great.

**Review by sareh — COMMENTED** [review]
👍 This works great locally. Just a couple of styling comments.

**Review by sareh — DISMISSED** [review]
👍 Looks great! Thanks for the first AMP PR :)

**Review by sareh — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Comment by sareh** [comment]
This resolved #441

**Pull request merged**